tests/plugins/widget/manual/widgetwithspaces.html

```diff
@@ -0,0 +1,25 @@
+<div id="editor"></div>
+
+<script>
+	CKEDITOR.replace( 'editor', {
+		extraAllowedContent: 'span',
+		extraPlugins: 'spanwidget',
+		allowedContent: true
+	} );
+
+	CKEDITOR.plugins.add( 'spanwidget', {
+		requires: 'widget',
+		init: function ( editor )	{
+			editor.widgets.add( 'spanwidget', {
+				editables: {
```

Comment:
Only block widgets support `editables`. Even if you define it like that, the inline widget or its part will not be editable. You can remove the `editables` definition as it does not change anything here (you can see by running this test and trying to edit the `span` element inside widget).